src/js/App/RootApp.js

```diff
@@ -89,11 +89,16 @@ ShieldedRoot.displayName = 'ShieldedRoot';
 const RootApp = ({ activeApp, activeLocation, appId, config, pageAction, pageObjectId, globalFilterHidden }) => {
   const scalprum = useScalprum(config);
   const hideNav = useSelector(({ chrome: { user } }) => !user);
-  const isLanding = useSelector(({ chrome }) => chrome?.appId === 'landing');
+  const { pathname } = useLocation();
+  /**
+   * Using the chrome landing flag is not going to work because the appId is initialized inside the app.
+   * We need the information before anything is rendered to determine if we use root module or render landing page.
+   * This will be replaced once we can use react router for all pages. Landing page will have its own route.
+   */
+  const isLanding = pathname === '/';
```

Comment:
What about beta pathname?

Reply:
It's taken care of by the router basename.